### PR TITLE
sysdeps: add aero sysdep

### PR DESCRIPTION
**Pull request by Andy-Python-Programmer**
This pull request adds the [aero](https://github.com/Andy-Python-Programmer/aero) sysdep :^)

Signed-off-by: Andy-Python-Programmer <andypythonappdeveloper@gmail.com>

### CODE REVIEW

File: sysdeps/aero/include/abi-bits/auxv.h [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../abis/lemon/auxv.h
```
[the remark this answers is not in the document]
Comment (Andy-Python-Programmer) [reply sysdeps/aero/include/abi-bits/auxv.h]: Luckily they were same so it worked, but nice catch :^)

**Review by Geertiebear — COMMENTED** [review]
LGTM in general but could you please rename the ``.cc`` files to ``.cpp``? That is more in-line with mlibc C++ style.

**Comment by Andy-Python-Programmer** [comment]
> LGTM in general but could you please rename the `.cc` files to `.cpp`? That is more in-line with mlibc C++ style.

done :^)

**Comment by avdgrinten** [comment]
It would be a good idea to add `aero` to the GHA CI as well.

**Comment by Andy-Python-Programmer** [comment]
Don't know how

**Comment by ArsenArsen** [comment]
https://github.com/managarm/mlibc/blob/master/.github/workflows/ci.yml#L51

**Comment by Andy-Python-Programmer** [comment]
done :^)

File: sysdeps/aero/generic/filesystem.cpp [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+    if (optional_action)
+        mlibc::infoLogger()
+            << "mlibc: warning: sys_tcsetattr ignores optional_action"
+            << frg::endlog;
+
+    int result;
+
+    // 0x5404 = TCSETSF (using this workaround since the headers don't define
+    // it)
+    if (int e = sys_ioctl(fd, 0x5404, (void *)attr, &result); e)
```
Comment (Dennisbonke) [inline sysdeps/aero/generic/filesystem.cpp:136]: After #368, you can drop the hardcoded hack and use the actual define.

Comment (Geertiebear) [reply sysdeps/aero/generic/filesystem.cpp:136]: #368 is merged.

Comment (Andy-Python-Programmer) [reply sysdeps/aero/generic/filesystem.cpp]: done :^)

**Pull request merged**